### DISCUSSION TITLE
Chromium Failed to launch browser

### DISCUSSION THREAD

**Discussion by lucy7099** [percy/cli#553] [Support] [answered]
hi when running the percy CLI snapshot tests as part of gitlab ci when i merge to master I am facing this error:
has anyone come across this before and how did you solve it? can't seem to find anything useful on google.

thanks

```
$ npx percy snapshot src/test/percy/snapshots.yml
[percy] Downloading Chromium 885264...
[percy] Successfully downloaded Chromium 885264
[percy] Error: Failed to launch browser. 
/builds/zSHei7dC/0/ticketed/website/node_modules/@percy/core/.local-chromium/885264/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory
'
Cleaning up file based variables
00:00
ERROR: Job failed: command terminated with exit code 1

```

**Comment by Robdel12** [comment]
Hey @lucy7099! That means the docker container you're using in CI doesn't have the right dependencies installed in order to launch a browser. You'll want to change the container image to something that has the ability to launch a browser. For example, CircleCI provides images that can launch browsers with `-browsers` in their name (i.e. `14.17.6-browsers`): https://circleci.com/developer/images/image/cimg/node

**Comment by lucy7099** [comment, marked as answer]
Tried using `cimg/node:16.8.0-browsers` however I have `yarn install` issues now. This was not the case before.
It says:

*Make sure your network and proxy settings are correct.*

My gitlabci configuration

```
percy:
  image: cimg/node:16.8.0-browsers
  variables:
    PERCY_TOKEN: "$PERCY_TOKEN"
  stage: percy
  script:
    - yarn install
    ...
```

**Reply by lucy7099** [reply]
manually installing these libraries for chromium in my gitlab set up worked for now 
`- apt-get install -y libnss3-tools libatk-bridge2.0-0 libdrm2 libxkbcommon0 libgbm-dev libasound2 libxshmfence-dev`
